org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/loader/AntaresLoaderDebriefLite.java

```diff
@@ -103,9 +106,17 @@ public void actionPerformed(ActionEvent e) {
 				antaresImporter.setTrackName(nameOfTheTrackTextField.getText());
 				try {
 					antaresImporter.importThis(file.getName(), new FileInputStream(file));
+					if (!antaresImporter.getErrors().isEmpty()) {
```

Comment:
this is very minor @saulhidalgoaular - but wouldn't it be better to just call `antaresImporter.getErrors()` once, and use the results object?  (we currently call it twice).